src/container-manager/docker-files/lumino-node/assets/server.json5

```
@@ -0,0 +1,40 @@
+{
+  libp2p: {
+    addresses: {
+      listen: [
+        "/ip4/127.0.0.1/tcp/6010",
+        "/ip4/127.0.0.1/tcp/6011/ws"
+      ]
+    },
+    config: {
+      peerDiscovery: {
+        bootstrap: {
+          enabled: false
+        }
+      }
+    }
+  },
+  loadPrivKeyFromFile: true,
+  key : {
+    createNew: false,
+    password:"infuy",
+    openSSL:true,
+    privateKeyURLPath:"file:////root/.rif-comms/server.der",
+    type: "DER"
+  },
+  rooms: ["0xtestroom", "0xtestroom6", "0xtestroom3"],
```

We don't need the test rooms
```suggestion
  rooms: ["0xtestroom", "0xtestroom6", "0xtestroom3"],
  rooms: []
```

---

I didn't understood the change. Are you sure it's fine? i mean why you just not remove the romms node then? is it empty? what value should be there?